src/commands/cmd_context.h

```diff
@@ -39,8 +41,23 @@ RedisModuleCtx *CommandCtx_GetRedisCtx
 	CommandCtx *qctx
 );
 
+// Get blocking client.
+RedisModuleBlockedClient *CommandCtx_GetBlockingClient(
+	const CommandCtx *qctx
+);
+
 // Get GraphContext.
-GraphContext* CommandCtx_GetGraphContext
+GraphContext *CommandCtx_GetGraphContext
+(
+	const CommandCtx *qctx
+);
+
+// Get command name.
+const char *CommandCtx_GetCommandName(
+	const CommandCtx *qctx
+);
+
+const char *CommandCtx_GetQuery
```

Comment:
Please be consistent with the position of the opening parenthesis

Reply:
fixed